website/docs/r/storage_table.html.markdown

```diff
@@ -68,6 +68,8 @@ In addition to the Arguments listed above - the following Attributes are exporte
 
 * `id` - The ID of the Table within the Storage Account.
 
+* `resource_manager_id` - The ID of the Table in the Resource Manager format. Useful for cases like IAM that do not use the Data Plane ID.
```

Comment:
We can match the description with how we've done it in other resources. I'd also caution against specifying how to use this as the docs can get very clogged up if we provided helpful tips on most things

```suggestion
* `resource_manager_id` - The Resource Manager ID of this Storage Table.
```